geoviews/tests/plotting/mpl/test_chart.py

```diff
@@ -54,7 +54,7 @@ def test_windbarbs_dataset(self):
         np.testing.assert_almost_equal(mpl_barbs.v.data, V.T.flatten())
 
     def test_windbarbs_from_uv(self):
-        x = np.linspace(-1, 1, 4)
+        x = np.linspace(-1, 1, 2)
```

Comment:
```suggestion
        x = np.array([-1, 1])
```

Nit again.

Is there a reason for the change in the input array?

Reply:
I was testing why it was failing so I reduced the size so I can compare & contrast more easily; I changed it back.